liblangutil/Exceptions.h

```diff
@@ -70,6 +70,7 @@ struct ErrorId
 {
 	unsigned long long error = 0;
 	bool operator==(ErrorId const& _rhs) const { return error == _rhs.error; }
+	bool operator!=(ErrorId const& _rhs) const { return !(*this == _rhs); }
```

Comment:
Why not:
```suggestion
	bool operator!=(ErrorId const& _rhs) const { return error != _rhs.error; }
```

It seems to be more readable.

Reply:
I prefer the original version. It keeps the main logic in a single function

Reply:
I think this is needless complication given these are one expressions each.

Reply:
Modularity.

Reply:
Sure, but in my opinion it is way less readable. I would agree to the modularity if this would be a large class, but it is a **4 line struct** where this is not warranted.